LDAR_Sim/src/programs/site_level_method.py

```diff
@@ -88,13 +88,13 @@ def __init__(
             self._inst_threshold = float("inf")
         self._follow_up_schedule: FollowUpMobileSchedule = follow_up_schedule
         self._detection_count = 0
-        self._small_window = properties[pdc.Method_Params.ROLLING_AVRG][
-            pdc.Method_Params.SMALL_WINDOW
-        ]
-        self._large_window = properties[pdc.Method_Params.ROLLING_AVRG][
-            pdc.Method_Params.LARGE_WINDOW
-        ]
         if self._deployment_type == pdc.Deployment_Types.STATIONARY:
```

Comment:
Can we add a try catch for key error that could occur if one of these are missing and print out an error?

Reply:
Also for the mobile only parameters can we do something similar?

Reply:
That won't ever happen due to the default parameters existing.